recipes/aravis/all/conanfile.py

```diff
@@ -91,8 +105,7 @@ def build_requirements(self):
         self.tool_requires("glib/<host_version>")
         if not self.conf.get("tools.gnu:pkg_config", check_type=str):
             self.tool_requires("pkgconf/1.9.5")
-        if self.options.introspection:
-            self.tool_requires("gobject-introspection/1.72.0")
```

Comment:
Was this library being a tool requires initially a bug? Or what's the reasonale behind this change?